### DISCUSSION TITLE
Search doesn’t work in Issues

### DISCUSSION THREAD

**Discussion by AnaCIDA** [community/community#45383] [Projects and Issues]
When searching for an issue on iOS app, search doesn't work (shows zero results although the searched issues definitely exist and come up in search results in web version). Tested on two iPhones.

**Comment by jge162** [comment]
Any GitHub related question? Maybe reach out to Apple support for your iphone issue. Sorry to hear you’re having trouble.

**Reply by AnaCIDA** [reply]
GitHub client on iOS is not developed by Apple, therefore Apple support will not be able to address the issue. It's a GitHub app issue.

**Reply by jge162** [reply]
I see what you are saying now. (GitHub Issues). It seemed like you meant iOS issue. Understood. Hopefully the developer can comment here for you. Bests
